cryptography/2.0/modules/ROOT/pages/cryptography-module-upgrade.adoc

```diff
@@ -105,6 +106,52 @@ Therefore, if you use BC version 1.80 or higher with Cryptography Module 2.0.0 i
 
 If you prefer a FIPS-compliant environment, use the `bc-fips` and `bcpg-fips` libraries.
 
+Example of Mule Maven Plugin configuration for the Cryptography Module using Bouncy Castle JDK18on Dependencies Version 1.80:
+
```

Comment:
I just spotted there is another error on the page - in line 83, could you fix "JDK18on" to be "JDK 18 on"